### PR TITLE
[#12048] Fix GetSessionResponseStatsActionIT

### PR DESCRIPTION
**Pull request by domoberzin**
Part of #12048 

Fixes the failing assertions in PR #12753

**Outline of Solution**

### CODE REVIEW

File: src/main/java/teammates/sqllogic/core/FeedbackSessionsLogic.java [hunk 1]
```diff
@@ -169,6 +171,8 @@ public Set<String> getGiverSetThatAnsweredFeedbackSession(String feedbackSession
         FeedbackSession feedbackSession = fsDb.getFeedbackSession(feedbackSessionName, courseId);
 
         Set<String> giverSet = new HashSet<>();
+
+        // TODO: to check if this is the correct way to get the giver set
```
Comment (ziqing26) [inline src/main/java/teammates/sqllogic/core/FeedbackSessionsLogic.java:175]: Let's remove this line
```suggestion

```

**Review by ziqing26 — APPROVED** [review]
LGTM! Thanks for the fix!

File: src/main/java/teammates/storage/sqlapi/FeedbackResponsesDb.java [hunk 2]
```diff
@@ -164,6 +164,20 @@ public boolean areThereResponsesForQuestion(UUID questionId) {
         return !HibernateUtil.createQuery(cq).getResultList().isEmpty();
     }
 
+    /**
+     * Get responses responses for a question.
```
Comment (cedricongjh) [inline src/main/java/teammates/storage/sqlapi/FeedbackResponsesDb.java:168]: nit: double response

File: src/main/java/teammates/sqllogic/core/FeedbackSessionsLogic.java [hunk 3]
```diff
@@ -495,4 +513,45 @@ public List<FeedbackSession> getFeedbackSessionsWhichNeedOpenEmailsToBeSent() {
...
+
+    /**
+     * Gets the expected number of submissions for a feedback session.
+     */
+    public int getExpectedTotalSubmission(FeedbackSession fs) {
+        int expectedTotal = 0;
+        List<FeedbackQuestion> questions = fqLogic.getFeedbackQuestionsForSession(fs);
+        if (fqLogic.hasFeedbackQuestionsForStudents(questions)) {
+            expectedTotal += usersLogic.getStudentsForCourse(fs.getCourse().getId()).size();
+            // usersLogic.getUn
```
Comment (cedricongjh) [inline src/main/java/teammates/sqllogic/core/FeedbackSessionsLogic.java:525]: nit: remove this comment

**Review by cedricongjh — APPROVED** [review]

**Pull request merged**